src/AbpDevTools/Commands/DatabaseDropCommand.cs

```diff
@@ -0,0 +1,81 @@
+using System.Diagnostics;
+using AbpDevTools.Notifications;
+using CliFx.Infrastructure;
+
+namespace AbpDevTools.Commands;
+
+[Command("database-drop", Description = "Drops all databases in the working directory")]
+public class DatabaseDropCommand : ICommand
+{
+    [CommandParameter(0, IsRequired = false, Description = "Working directory to search for EntityFramework projects. Default: . (Current Directory)")]
+    public string WorkingDirectory { get; set; }
+    
+    [CommandOption("force", 'f')]
+    public bool Force { get; set; }
+    
+    protected readonly INotificationManager notificationManager;
+
+    public DatabaseDropCommand(INotificationManager notificationManager)
+    {
+        this.notificationManager = notificationManager;
+    }
+
+    public async ValueTask ExecuteAsync(IConsole console)
+    {
+        if (string.IsNullOrEmpty(WorkingDirectory))
+        {
+            WorkingDirectory = Directory.GetCurrentDirectory();
+        }
+
+        var efCoreProjects = Directory.EnumerateFiles(WorkingDirectory, "*.csproj", SearchOption.AllDirectories)
+            .Where(x => x.EndsWith("EntityFrameworkCore.csproj"))
+            .Select(x => new FileInfo(x))
+            .ToList();
+            
+        var cancellationToken = console.RegisterCancellationHandler();
```

Comment:
When a cancellation requested by `CTRL` + `C`, the process can stay hanging. We should kill running process with it's children processes together. Here is an example:
https://github.com/enisn/AbpDevTools/blob/f22b39401acca0899108aff2c7346b9a598f69f3/src/AbpDevTools/Commands/BuildCommand.cs#L115

Reply:
Ok, they're running one by one, and throıwn exception by cancellationtoken prevents handling processes ✅ 👍